UAMX3-65 point all builds to UAM spec (not UTM)

fshihames opened the pull request:
For UAMX3-65 point all builds to UAM spec (not UTM)
This is to clean up scripts/code/usage of the following:
fims-ops, urep, uss_disc, uss_resources, activemq

I assume that we will not keep uss_resources. If the assumption is incorrect, the changes should be adjusted accordingly.

This check-in of change involve the following repos, all in the branch of UAMX3-65:
fims-deployment, nuss, utm-commons, uam-apis

The scope of this check-in involves many files, especially many are related to deployment. Some files have legacy of history which I am very ignorant of.  I feel I am just facilitating the task, I need the helps of you, the master minds.  I don't have a good way to test it. I suggest review them carefully. Once ready, we may want to designate a chunk of time to deploy them and fix them in walking through the deployment/delivery process, right?

fshihames committed:
UAMX3-65 point all builds to UAM spec (not UTM)
    More on cleaning up scripts/code/usage of the following:
    fims-ops, urep, uss_disc, uss_resources, activemq

issmith1 reviewed (approved):
i see you have changed the Parameters into all Jenkins jobs from v4-draft to master.
Excellent job, Fu Tai!  this was a big project with lots of little moving parts.

fshihames committed:
UAMX3-65 point all builds to UAM spec (not UTM)
        More on cleaning up scripts/code/usage of the following:
        fims-ops, urep, uss_disc, uss_resources, activemq
